wpilibc/athena/src/Vision/BaeUtilities.cpp

```diff
@@ -101,32 +102,32 @@ void dprintf(const char* tempString, ...) {
   va_end(args);
 
   /* Format output statement */
-  std::stringstream ss;
-  ss << std::setfill('0') << std::setw(4);
-  ss << "[" << filename << ":" << functionName << "@" << line_number << "] ";
+  std::ostringstream oss;
```

Comment:
It pads the output with a lot of zeroes, so I figured it should stay for output readability, not that it matters since this file has been removed...